server.js

```diff
@@ -3,23 +3,26 @@ var io = require('socket.io')(port);
 var ClientManagerService = require('./app/services/ClientManagerService');
 var LobbyManagerService = require('./app/services/LobbyManagerService');
 
-console.log("League Sandbox Lobby Server");
 console.log("---------------------------");
+console.log("League Sandbox Lobby Server");
```

Comment:
In my opinion the old format for this worked more on par with most written content, as in
```
{title}
{section division}
{content}
```
now you have
```
{section division}
{title}
{content}
{section division}
```
but that's a pretty minor complaint, so no absolute need to change